.travis.yml

```diff
@@ -1,6 +1,7 @@
 sudo: false
 language: java
 jdk: oraclejdk8
+dist: trusty
 cache:
```

Comment:
Why it is here? This line exists in both two branches。

Reply:
I don't know, but the JDK is out of date, AFAIK. It needs to be `openjdk8`.

Reply:
Didn't you add it?

Reply:
Yeah, it is out of date. This line is another way to pass the travis building. The build failure is from the source code. I am still working on it.